WordPress/Classes/ViewRelated/Reader/Select Interests/ReaderSelectInterestsViewController.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+
+class ReaderSelectInterestsViewController: UIViewController {
+    private struct Constants {
+        static let reuseIdentifier = ReaderInterestsCollectionViewCell.classNameWithoutNamespaces()
+        static let interestsLabelMargin: CGFloat = 10
+
+        static let cellCornerRadius: CGFloat = 8
+        static let cellSpacing: CGFloat = 6
+        static let cellHeight: CGFloat = 40
+    }
+
+    // MARK: - IBOutlets
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var subTitleLabel: UILabel!
+    @IBOutlet weak var collectionView: UICollectionView!
+    @IBOutlet weak var buttonContainerView: UIView!
+    @IBOutlet weak var nextButton: FancyButton!
+
+    // MARK: - Mock Data Source
+    private let fakeDataSource = InterestsDataSource(fileName: "interests.json")
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureCollectionView()
+        applyStyles()
+    }
+
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        let layout = collectionView.collectionViewLayout as! ReaderInterestsCollectionViewFlowLayout
+        layout.invalidateLayout()
+    }
+
+    // MARK: - Private Methods
+    private func configureCollectionView() {
+        let nib = UINib(nibName: String(describing: ReaderInterestsCollectionViewCell.self), bundle: nil)
+        collectionView.register(nib, forCellWithReuseIdentifier: Constants.reuseIdentifier)
+
+        guard let layout = collectionView.collectionViewLayout as? ReaderInterestsCollectionViewFlowLayout else {
+            return
+        }
+
+        layout.itemSpacing = Constants.cellSpacing
+        layout.cellHeight = Constants.cellHeight
+    }
+
+    private func applyStyles() {
+        let styleGuide = ReaderInterestsStyleGuide.self
+        styleGuide.applyTitleLabelStyles(label: titleLabel)
+        styleGuide.applySubtitleLabelStyles(label: subTitleLabel)
+        styleGuide.applyNextButtonStyle(button: nextButton)
+
+        buttonContainerView.backgroundColor = ReaderInterestsStyleGuide.buttonContainerViewBackgroundColor
+    }
+}
+
+// MARK: - UICollectionViewDataSource
+extension ReaderSelectInterestsViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return fakeDataSource.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell: ReaderInterestsCollectionViewCell = collectionView.dequeueReusableCell(withReuseIdentifier: Constants.reuseIdentifier, for: indexPath) as! ReaderInterestsCollectionViewCell
```

Comment:
I prefer guard instead of forced unwrapping

Reply:
Done in: https://github.com/wordpress-mobile/WordPress-iOS/pull/14402/commits/f6de145dc492798e13896f7c9f26585a834d933f